test/LndClient/RPCSpec.hs

```diff
@@ -100,6 +105,13 @@ merchantEnv env = do
           }
     }
 
+btcClient :: IO Client
+btcClient = do
+  user <- Char8.pack . btcRpcUser <$> btcConfig
```

Comment:
`btcConfig` there is IO type.
It means that it will parse system variables 3 times, but you really need only 1 time
Then you can extract fields and pack them how you want, because field accessors and pack are pure functions (you can use for example `let` if you want to bind them to some variables)

Reply:
This one is good one, managed to implement it, please check, maybe some simpler way is possible.